src/Microsoft.Diagnostics.Monitoring.WebApi/ParameterCapturing/CapturedParametersTextFormatter.cs

```diff
@@ -20,7 +20,7 @@ protected override async Task WriteCapturedMethodAsync(CapturedMethod capture, C
         {
             StringBuilder builder = new();
 
-            builder.AppendLine(FormattableString.Invariant($"[{capture.CapturedDateTime}][thread {capture.ThreadId}] {GetValueOrUnknown(capture.ActivityId)}[format: {capture.ActivityIdFormat}]"));
+            builder.AppendLine(FormattableString.Invariant($"[{capture.CapturedDateTime}][thread {capture.ThreadId}] {GetValueOrUnknown(capture.ActivityId)}[format: {capture.ActivityIdFormat}][methodToken: {capture.MethodToken}][moduleVersionId: {capture.ModuleVersionId}]"));
```

Comment:
We probably don't care about the mvid/methodtoken when doing text formatting, but the feature is experimental so we can always change this in the future

Reply:
Agreed, text output shouldn't contain tokens in it.